index.html

```diff
@@ -0,0 +1,174 @@
+            </div>
+        </div>
+    </section>
+    <section id="program" class="flex-col">
+        <h2>Main Program</h2>
+        <hr class="dash">
+        <div class="program-items">
+            <div>
+                <img src="asset/speaker.svg" alt="speaker">
+                <h6>Lecture</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/exhibit.svg" alt="speaker">
+                <h6>OV Exhibit</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/forum.svg" alt="speaker">
+                <h6>Forum</h6>
+                <p>
+                    get opportunities to network with CC affiliates around the world, also after the summit.
+                </p>
+            </div>
+            <div>
+                <img src="asset/workshop.svg" alt="speaker">
+                <h6>Workshop</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/network.svg" alt="speaker">
+                <h6>OG Ignite</h6>
+                <p>
+                    get opportunities to network with CC affiliates around the world, also after the summit.
+                </p>
+            </div>
+        </div>
+        <button type="button" id="join-btn" class="btn">Join OV Gaming Summit 2015</button>
+        <a href="#">See the whole program</a>
+    </section>
+    <section id="speaker-section" class="flex-col">
+        <div class="flex-col">
+            <h2>Featured Speakers</h2>
+            <hr class="dash">
+        </div>
+        <div id="speaker-card-container"></div>
+    </section>
+    <section id="desktop-sponsors">
+        <div id="sponsor-wrapper">
+            <h3>Partner</h3>
+            <hr class="dash">
+            <div>
+                <img class="sponsors-img" src="asset/logitech.png" alt="logitech">
+                <img src="asset/redbullcom-logo.svg" alt="redbull">
+                <img src="asset/steam.png" alt="steam">
+                <img src="asset/riot.png" alt="riot">
+                <img class="sponsors-img" src="asset/twitch.png" alt="twitch">
+            </div>
+        </div>
+    </section>
+    <footer id="footer-desktop" class="footer-section">
+        <div class="footer-container flex-row">
+            <div class="logo-img-container">
+                <img src="asset/fire.png" alt="logo">
+                <h2>
+                    OG Gaming
+                    <br> 
+                    Summit 2023
+                </h2>
+            </div>
```

Comment:
- [ ] Well done, you adjust the things asked by your previous code reviewer but still I noticed that you need to make a few more adjustments in your mobile design. Your footer is looking a little bit *OFF* as compared to the design given on the *Behance*, which makes your design looks unprofessional. Please fix the issue in order to move on.

### Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Behance Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-24 09-08-18](https://github.com/over-geek/capstone-project-2/assets/66476329/3fde3003-23d4-4a0d-9706-818c46eab68a)

</td>

<td>

![Screenshot from 2023-05-24 09-08-28](https://github.com/over-geek/capstone-project-2/assets/66476329/891a05d9-3037-4253-be6d-b0ca92584691)

</td>
</tr>
</table>